.circleci/config.yml

```diff
@@ -183,6 +195,9 @@ workflows:
   workflow:
     jobs:
       - compile
+      - build_release:
```

Comment:
I'd expect this farther down next to build_instrumented to match the ordering in top. Feel free to ignore.

Reply:
Yeah good point. Will change.